test/groovy/PiperStageWrapperTest.groovy

```diff
@@ -161,6 +162,7 @@ class PiperStageWrapperTest extends BasePiperTest {
         helper.registerAllowedMethod('load', [String.class], {
             return helper.loadScript('test/resources/stages/test_old_extension.groovy')
         })
+        nullScript.env.PIPER_DISABLE_EXTENSIONS = 'false'
```

Comment:
Not setting the env var should not fail the step execution! It the test fail due to this we need to change the code, not the test.